documentation/ResetComputer.md

```diff
@@ -0,0 +1,124 @@
+                    "http://registry\\.gpii\\.net/common/magnification": "mag-factor",
+                    "http://registry\\.gpii\\.net/common/showCrosshairs": "show-cross-hairs",
+                    "transform": [
+                        {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInputPath": "screen-position",
+                            "defaultOutputPath": "http://registry\\.gpii\\.net/common/magnifierPosition",
+                            "match": {
+                                "full-screen": "FullScreen",
+                                "left-half": "LeftHalf",
+                                "right-half": "RightHalf",
+                                "top-half": "TopHalf",
+                                "bottom-half": "BottomHalf"
+                            }
+                        },
+                        {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultOutputPath": "http://registry\\.gpii\\.net/common/tracking",
+                            "defaultInputPath": "mouse-tracking",
+                            "match": {
+                                "centered": "mouse"
+                            }
+                        }
+                    ]
+                },
+                "settings": {
+                    "mag-factor": {
+                        "type": "DELETE"
+                    },
+                    "screen-position": {
+                        "type": "DELETE"
+                    }
+                }
+            }
+        },
+        "launchHandlers": {
+            "launcher": {
+                "type": "gpii.gsettings.launch",
+                "options": {
+                    "schema": "org.gnome.desktop.a11y.applications",
+                    "key": "screen-magnifier-enabled"
+                },
+                "settings": {
+                    "running": {
+                        "value": false
+                    }
+                }
+            }
+        },
+        "update": [
+            "settings.configuration"
+        ],
+        "isInstalled": [
+            {
+                "type": "gpii.packageKit.find",
+                "name": "gnome-shell"
+            }
+        ],
+        "active": true
+    }
+}
+```
+
+When GPII starts, it copies `defaultSettings.json5` from `testData/solutions` directory to the GPII settings directory
+if it hasn't been copied. GPII reads the default settings from `defaultSettings.json5` located at the GPII settings
+directory. Users are invited to edit the default settings file at the settings directory instead of at
+`testData/solutions` directory to keep the code base clean and consistent.
+
+## Reset the Computer
+
+Once the default settings file is ready, the reset can be initiated by sending http requests to APIs provided by the
+local flow manager. These APIs include:
```

Comment:
Rather than "include", say "are" as above since include suggests that there might be more